apps/store/modules/page-decorators.js

```diff
@@ -601,6 +601,14 @@ var pageDecorators = {};
         page.searchHistory = {};
         page.searchHistory.queries = userApi.getSearchHistory(ctx.session, ctx.assetType);
     };
+    pageDecorators.taxonomyAvailability = function(ctx, page, utils) {
+        var app = require('rxt').app;
```

Comment:
Thank you Sameera for suggestion. But app.isTaxonomyEnabled() code block has been refactored with multiple taxonomy. Its not available anymore.